projects/aca-playwright-shared/src/page-objects/components/aca-header.component.ts

```diff
@@ -38,9 +39,15 @@ export class AcaHeader extends BaseComponent {
   constructor(page: Page) {
     super(page, AcaHeader.rootElement);
   }
+  public matMenu = new MatMenuComponent(this.page);
 
   async clickViewerMoreActions(): Promise<void> {
     await this.moreActionsButton.waitFor({ state: 'attached' });
     await this.moreActionsButton.click();
   }
+
+  async selectCreateFolderFromTemplate(): Promise<void> {
```

Comment:
name is a bit confusing as the function does not select anything... maybe rename to "select" to "click" ?

Reply:
name was change as suggested